Ginger/GingerCore/ALM/QCCore.cs

```diff
@@ -78,16 +78,9 @@ public override ObservableList<Activity> GingerActivitiesRepo
             set { ImportFromQC.GingerActivitiesRepo = value; }
         }
 
-        public override ObservableList<ExternalItemFieldBase> GetALMItemFields(BackgroundWorker bw, bool online, bool useREST)
+        public override ObservableList<ExternalItemFieldBase> GetALMItemFields(BackgroundWorker bw, bool online, ALM_Common.DataContracts.ResourceType resourceType)
         {
-            if (!useREST)
-            {
-                return ImportFromQC.GetALMItemFields();
-            }
-            else
-            {
-                return ImportFromQC.GetQCEntityFieldsREST(ALM_Common.DataContracts.ResourceType.DEFECT); // currently DEFECTS supported. May be extended if needed
-            }
+            return ImportFromQC.GetALMItemFields();
         }
```

Comment:
You not supposed to pass the resourceType to the GetALMItemFields() method here?

Reply:
This is QCCore. They implement it without resource type. Oleg add some bool as you can see and use hard coded resource defects. I correct it. Now it works fine. If someone need to implement with resource in OTA now he can. In the QCRestCore when i need to go only to 1 resource type i use it